helmfile.d/values/networkpolicies/workload/kyverno.yaml.gotmpl

```diff
@@ -0,0 +1,33 @@
+policies:
+  kyverno:
+    all-deny:
+      podSelectorLabels: {}
+      egress: {}
+      ingress: {}
+
+    kyverno-admission-controller:
+      podSelectorLabels:
+        app.kubernetes.io/instance: kyverno
+        app.kubernetes.io/component: admission-controller
+      egress:
+        - rule: egress-rule-apiserver
+        - rule: egress-rule-dns
+        - name: egress-rule-image-registry
+          peers:
+            - cidr: 0.0.0.0/0
+          ports:
+            - tcp: 443
+            - tcp: 80
```

Comment:
This should be exposed, with all but the air-gapped flavour containing this as default.